code/sorting/bogo_sort/bogo_sort.cpp

```diff
@@ -0,0 +1,42 @@
+/* Part of Cosmos by OpenGenus Foundation */
+// C++ implementation of BogoSort by ldaw
+
+#include <algorithm>
+#include <iostream>
+#include <vector>
+#include <cstdlib>
+#include <ctime>
+
+bool isSorted(std::vector<int> &v)
+{
+	int i = v.size();
+	while (--i > 0)
+		if (v[i] < v[i-1])
+			return false;
+	return true;
+}
+
+void shuffle(std::vector<int> &v)
+{
+	for (int i = 0; i < v.size(); i++)
+		std::swap( v[i], v[ rand() % v.size() ] );
+}
+
+void bogoSort(std::vector<int> &v)
+{
+	while ( !isSorted(v) )
+		shuffle(v);
+}
+
+int main()
+{
+	std::srand(time(NULL));
+	std::vector<int> v = {2, 4, 1, 3, 6, 7, 8, 5};
```

Comment:
Initializer list is a C++11 feature. In these cases it would be helpful to provide a comment specifying the standard requirement.